tools/KeyfileIOUtils.cpp

```diff
@@ -64,6 +73,18 @@ static void validateRSAPrivateKey(const std::string& key) {
   if (!std::regex_match(key, std::regex("[0-9A-Fa-f]+"))) throw std::runtime_error("Invalid RSA private key: " + key);
 }
 
+static void validateEdDSAPublicKey(const std::string& key) {
+  if (!(key.length() == EdDSAPublicKeyByteSize * 2) && (std::regex_match(key, std::regex("[0-9A-Fa-f]+")))) {
+    throw std::runtime_error("Invalid EdDSA public key: " + key);
+  }
+}
+
+static void validateEdDSAPrivateKey(const std::string& key) {
+  if (!(key.length() == EdDSAPrivateKeyByteSize * 2) && (std::regex_match(key, std::regex("[0-9A-Fa-f]+")))) {
```

Comment:
same as validateEdDSAPublicKey

Reply:
Changes in https://github.com/arc-vmware/concord-bft/pull/6/commits/87f4edd0f4f5d72b422cef50f4a8eeeee8b28bc2